code/modules/clothing/gloves/tacklers.dm

```diff
@@ -0,0 +1,99 @@
+/obj/item/clothing/gloves/tackler
+	name = "gripper gloves"
+	desc = "Special gloves that manipulate the blood vessels in the wearer's hands, granting them the ability to launch headfirst into walls."
+	icon_state = "tackle"
+	item_state = "tackle"
+	transfer_prints = TRUE
+	cold_protection = HANDS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
+	resistance_flags = NONE
+	var/datum/component/tackler
+	// I already covered these vars in components/tackle.dm, but I'll be nice and give you the rundown here too
```

Comment:
you can link to the component here with [ ]

Reply:
for each var anyway, you can just be like `/// see: [/datum/component/tackler/var/varname]`